docs/ProfilerOptions.md

```diff
@@ -56,6 +56,7 @@ The below options are `action`s for async-profiler and common for both `asprof`
 | `--libpath PATH`   | `libpath=PATH`    | Full path to `libasyncProfiler.so` (useful when profiling a container from the host).                                                                                                                                                                                                                                                                                                                                                                                                                                                       |
 | `--filter FILTER`  | `filter=FILTER`   | In the wall-clock profiling mode, profile only threads with the specified ids.<br>Example: `asprof -e wall -d 30 --filter 120-127,132,134 Computey`                                                                                                                                                                                                                                                                                                                                                                                         |
 | `--fdtransfer`     | `fdtransfer`      | Run a background process that provides access to perf_events to an unprivileged process. `--fdtransfer` is useful for profiling a process in a container (which lacks access to perf_events) from the host.<br>See [Profiling Java in a container](ProfilingInContainer.md).                                                                                                                                                                                                                                                                |
+| `--target-cpu`     | `target-cpu`      | In PerfEvents profiling mode, instruct the profiler to only sample threads running on the specified CPU, defaults to -1.<br>Example: `asprof --target-cpu 3`.                                                                                                                                                                                                                                                                                                                                                                               |
```

Comment:
`PerfEvents` -> `perf_events` everywhere in the docs

Reply:
7aaaa17575b5b6be9d2b2e204e650d05f77b7ee6